lib/mvi/service.rb

```diff
@@ -39,44 +39,10 @@ def find_profile(user)
         raw_response = perform(:post, '', create_profile_message(user), soapaction: OPERATIONS[:find_profile])
         MVI::Responses::FindProfileResponse.with_parsed_response(raw_response)
       end
-    rescue Faraday::ConnectionFailed => e
```

Comment:
`find_profile` resulted in an exception at this point. Caller can make the decision on how to handle the exception.

Reply:
And I think the basic idea is that we no longer fail gracefully. An error fetching MVI would result in the user serializer returning an error.

The only problem I see here is that the User object would be persisted, its session state would persisted, but the user would not be able to do anything while logged in because everytime user_serializer is fetched (VERY OFTEN), it will result in an exception. Perhaps we need to prevent login from persisting altogether in this scenario?